dashboard/postgres_replicator/logger.go

```diff
@@ -20,5 +20,5 @@ func NewLogger(tableName string) *Logger {
 
 // Errorf adds an "Error: " prefix to the format string
 func (l *Logger) Errorf(format string, v ...interface{}) {
-	l.Printf("Error: " + format)
+	l.Printf(fmt.Sprintf("Error: %s", format), v...)
```

Comment:
Can I ask why it didn't work before? It seems to me the two approach are equivalent.